### DISCUSSION TITLE
glances: command not found

### DISCUSSION THREAD

**Discussion by STUKguy** [nicolargo/glances#2727] [General]
**Describe the bug**
Installed using curl script but system reports not installed even after reboot

**To Reproduce**
1. curl -L https://bit.ly/glances | /bin/bash
2. sudo reboot
3. -bash: glances: command not found
![image](https://github.com/nicolargo/glances/assets/66088795/dda53999-7793-407d-a0b2-81c3285ef9af)

**Expected behavior**
Glances to be installed?

**Screenshots**

**Environement (please complete the following information)**
 - Operating System 

 No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

 - Glances & psutil versions: -bash: glances: command not found
 - How do you install Glances (Pypi package, script, package manager, source): curl -L https://bit.ly/glances | /bin/bash
 - Glances test (only available with Glances 3.1.7 or higher):

**Comment by peevishkiwi** [comment]
I have just discovered glances and also installed it on my raspberry pi, having the same issues as you. Did you find a solution? Did a lot of research but coul dnot find anything :/

**Comment by peevishkiwi** [comment]
@STUKguy : In case you still have this issue, i found a solution! 
The issue is that the folder, in which the scripts are installed / downloaded by PIP (/.local/bin by default) is "not looked" at when you try to run glances or any other app you download from pip. Therefore you need to add this folder the the .bashrc file. 

You can find examples and an explanation here: https://stackoverflow.com/questions/35898734/pip-installs-packages-successfully-but-executables-not-found-from-command-line
It is about another application in this discussion, but the root cause is the same.

1. open .bashrc file with a text editor of your choice, i used nano (nano .bashrc)
2. Scroll all the way down and add your pip download path  (/.local/bin in my case), i used this line from the stackoverflow discussion which worked fine: "export PATH="$HOME/.local/bin:$PATH".
3. Run "source ~/.bashrc" 
4. Try again to run any glances command. For me it worked afterwards!
